content/courses/onchain-development/intro-to-anchor.md

```diff
@@ -40,25 +45,31 @@ with writing a Solana program.
 
 Anchor uses macros and traits to generate boilerplate Rust code for you. These
 provide a clear structure to your program so you can more easily reason about
-your code. The main high-level macros and attributes are:
+your code and focus more on the functionality of your program. The main
+high-level macros and attributes are:
+
+> From here on out, you'll see a lot of Rust. We assume that you are familiar
+> with Rust, if not, we recommend you to check out the
+> [Rust Book](https://doc.rust-lang.org/book/).
 
-- `declare_id` - a macro for declaring the program’s onchain address
+- `declare_id!` - a macro for declaring the program’s onchain address
 - `#[program]` - an attribute macro used to denote the module containing the
-  program’s instruction logic
+  program’s instruction logic. This is where all the instruction handlers are
+  defined.
 - `Accounts` - a trait applied to structs representing the list of accounts
-  required for an instruction
+  required for an instruction.
 - `#[account]` - an attribute macro used to define custom account types for the
-  program
+  program.
 
 Let's talk about each of them before putting all the pieces together.
 
 ### Declare your program ID
 
-The `declare_id` macro is used to specify the onchain address of the program
-(i.e. the `programId`). When you build an Anchor program for the first time, the
-framework will generate a new keypair. This becomes the default keypair used to
-deploy the program unless specified otherwise. The corresponding public key
-should be used as the `programId` specified in the `declare_id!` macro.
+The `declare_id` macro sets the on-chain address of the Anchor program (i.e. the
```

Comment:
`onchain` is one word. https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#prose